gb-roms/src/controllers/mbc5.rs

```diff
@@ -1,19 +1,21 @@
+use super::Controller;
 use crate::header::size::{RamSize, RomSize};
 use gb_bus::{Address, Area, Error, FileOperation};
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
 use std::io::{self, Read};
 
-pub const MBC5_ROM_BANK_SIZE: usize = 0x4000;
-pub const MBC5_MAX_ROM_BANK: usize = 0x1FF;
-pub const MBC5_RAM_BANK_SIZE: usize = 0x2000;
-pub const MBC5_MAX_RAM_BANK: usize = 0x10;
-
 pub struct MBC5 {
-    rom_bank: Vec<[u8; MBC5_ROM_BANK_SIZE]>,
-    ram_bank: Vec<[u8; MBC5_RAM_BANK_SIZE]>,
+    rom_banks: Vec<[u8; MBC5::ROM_BANK_SIZE]>,
```

Comment:
The size of the array `r(o/a)m_banks` is only know at the initialization by reading the value `r(o/a)m_size` from the  header.
I choose `Vec` because of ease to use